packages/apps/frontera/src/routes/prospects/src/components/KanbanCard/KanbanCard.tsx

```diff
@@ -56,23 +57,21 @@ export const KanbanCard: React.FC<KanbanCardProps> = observer(
     const store = useStore();
     const navigate = useNavigate();
 
-    if (!card.value.organization) return null;
+    if (!card.value.metadata.id) return null;
 
-    const organization = store.organizations?.value.get(
-      card.value.organization?.metadata.id,
-    ) as OrganizationStore;
+    const organization = card.organization;
+    const logo = organization?.value.icon;
 
-    const logo = store.organizations?.value.get(
-      card.value.organization.metadata.id,
-    )?.value.logo;
+    const symbol = match(store.settings.tenant.value?.baseCurrency)
+      .with(Currency.Usd, () => '$')
```

Comment:
you can use `currencySymbol?.[store.settings.tenant.value?.baseCurrency]` from currencyOptions.ts instead